MiniTwit.Web.App/Views/Shared/_Layout.cshtml

```diff
@@ -14,13 +14,17 @@
     <h1>MiniTwit</h1>
     <div class="navigation">
         @if (SignInManager.IsSignedIn(User))
-        {
-            <a asp-area="" asp-controller="Home" asp-action="Index">my timeline</a>
-            @("|")
+        {    
+            //This method of putting the logout button in a form and wrapping all of the navigation in the form is a bit
+            //hacky, but its the easiest way to get them inline.
+            
+            <form method="post" asp-area="" asp-controller="Account" asp-action="LogOut">
+                <a asp-area="" asp-controller="Home" asp-action="Index">My Timeline</a>
+                @("|")
                 @*<a asp-area="" asp-controller="Home" asp-action="Public_TimeLine">public timeline</a>
                 @("|")*@
-
-            <a asp-area="" asp-controller="Account" asp-action="LogOut">Sign out @ViewData["username"]</a>
+                <input id="LogoutButton" type="submit" value="Logout"/>
```

Comment:
We have the option to use an invisible form that does not wrap the links, but is triggered through javascript instead.
This would cause some incompatability for browsers that have javascript disabled, but would instead remove the risk of logging users out if one of the links does not work.

Reply:
How could this cause users to logout if the link does not work?

Reply:
If the link is not valid (ie a #) the click event could bubble

Reply:
# Example
```html
<form>
    <a href="/some/address">This would work</a>
    <a href="#">This would submit the form</a>
</form>
```

Reply:
Alternatively we could just use a HttpGet and logout on that, thereby avoiding the POST problem.
Then we can can use a anchor <a> tag.